src/components/ui/areaCard.tsx

```diff
@@ -7,6 +7,8 @@ import { Icons } from "../icons";
 export interface AreaProps {
   id: string;
   name: string;
+  visible?: boolean;
+  toggleVisibility?: () => void;
```

Comment:
Added the optionality operator due to a typescript error about `visible` and `toggleVisibility` being missing from the `area` prop definition in `<CardWrapper />`